packages/client/wallets/smart-wallet/src/blockchain/wallets/passkey.ts

```diff
@@ -102,6 +106,32 @@ export class PasskeyAccountService {
             Authorization: `Bearer ${user.jwt}`,
         };
     }
+
+    private mapError(error: any, passkeyName: string) {
+        if (error.code === "ERROR_PASSTHROUGH_SEE_CAUSE_PROPERTY" && error.name === "NotAllowedError") {
+            return new PasskeyPromptError(passkeyName);
+        }
+
+        return error;
+    }
```

Comment:
Initially I tried also adding `error instanceof WebAuthnError` here first, which I imported in from the `@simpleWebAuthn` (what zerodev is using under the hood), but didn't work.